frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -430,6 +450,9 @@ export const surveyLogic = kea<surveyLogicType>([
             actions.setSurveyValue('conditions', NEW_SURVEY.conditions)
             actions.setSurveyValue('remove_targeting_flag', true)
         },
+        setSurveysOptIn: ({ optIn }) => {
+            actions.updateCurrentTeam({ surveys_opt_in: optIn })
+        },
```

Comment:
this will give the user a notice that their settings have been updated, which i think will be confusing because _they_ didn't update the setting, it's happening silently. I'd add a check in the `updateCurrentTeam` method to skip the notice if it's this setting, unless they are on the project settings page (where they would do it manually).

Reply:
removed~ since it's getting updated in the backend instead now!